api/src/main/java/com/qaengine/controllers/AnswerController.java

```diff
@@ -90,4 +90,32 @@ protected Answer downvoteAnswer(
         answer.setScore(answer.getScore() - 1);
         return answerRepository.save(answer);
     }
+
+
+    @CrossOrigin()
+    @PutMapping("questions/{questionId}/acceptAnswer/{answerId}")
+    protected Answer acceptAnswer(
+            @PathVariable Long questionId,
+            @PathVariable Long answerId
+    ) {
+        Answer answer = answerService.getAnswer(answerId);
```

Comment:
Just use answerId, later we can verify with authentication if user is allowed to accept the answer or not. Each answer has unique id anyway, and it's connected to only one question.